migration/idempotent/001-base.sql

```diff
@@ -1426,6 +1426,17 @@ COMMENT ON FUNCTION prom_api.set_default_chunk_interval(INTERVAL)
 IS 'set the chunk interval for any metrics (existing and new) without an explicit override';
 GRANT EXECUTE ON FUNCTION prom_api.set_default_chunk_interval(INTERVAL) TO prom_admin;
 
+CREATE OR REPLACE FUNCTION prom_api.get_default_chunk_interval()
```

Comment:
Maybe not for this PR, but I wonder if we should have separate default chunk intervals for metrics and traces?

Reply:
We do have separate default chunk intervals, but the chunk interval for traces is set when the trace tables are created. We don't provide any additional functions for handling the trace chunk interval. From a naming point of view, these functions should probably be renamed to `set/get_default_metric_chunk_interval`, but that's kinda out of scope of this change.